src/main/java/uk/gov/hmcts/ccd/CachingConfiguration.java

```diff
@@ -29,6 +29,7 @@ public Config hazelCastConfig() {
         config.setProperty("hazelcast.phone.home.enabled", "false");
         NetworkConfig networkConfig = config.setInstanceName("hazelcast-instance-ccd").getNetworkConfig();
         networkConfig.getJoin().getMulticastConfig().setEnabled(false);
+        networkConfig.getJoin().getAutoDetectionConfig().setEnabled(false);
```

Comment:
Required since we use a standalone instance
https://docs.hazelcast.org/docs/rn/index.html#bc-41